src/arguments/arguments.go

```diff
@@ -11,14 +11,17 @@ import (
 // ArgumentList contains all the arguments available for the F5 integration
 type ArgumentList struct {
 	sdkArgs.DefaultArgumentList
-	Hostname        string `default:"localhost" help:"The hostname or IP of the F5 BIG IP device to monitor."`
-	Port            int    `default:"443" help:"The port of the iControl API to connect to."`
-	Username        string `default:"" help:"The username to connect to the F5 API with."`
-	Password        string `default:"" help:"The password to connect to the F5 API with."`
-	Timeout         int    `default:"30" help:"The number of seconds to wait before a request times out."`
-	CABundleFile    string `default:"" help:"Alternative Certificate Authority bundle file"`
-	CABundleDir     string `default:"" help:"Alternative Certificate Authority bundle directory"`
-	PartitionFilter string `default:"[\"Common\"]" help:"JSON array of partitions to collect"`
+	Hostname          string `default:"localhost" help:"The hostname or IP of the F5 BIG IP device to monitor."`
+	Port              int    `default:"443" help:"The port of the iControl API to connect to."`
+	Username          string `default:"" help:"The username to connect to the F5 API with."`
+	Password          string `default:"" help:"The password to connect to the F5 API with."`
+	Timeout           int    `default:"30" help:"The number of seconds to wait before a request times out."`
+	CABundleFile      string `default:"" help:"Alternative Certificate Authority bundle file"`
+	CABundleDir       string `default:"" help:"Alternative Certificate Authority bundle directory"`
+	PartitionFilter   string `default:"[\"Common\"]" help:"JSON array of partitions to collect"`
+	AuthHost          string `default:"" help:"Hostname or IP address of the Big IQ auth token provider."`
```

Comment:
It appears that `AuthHost` and `AuthPort` were added without being used anywhere. Are these required for alternative login providers?

Reply:
Odd, I don't recall adding those two lines. Let me ask the F5 tech rep if it's needed.

Reply:
Our F5 contact says "Yes", we should parameterize AuthHost & AuthPort as this is not necessarily the iControl host/port.

Tests are failing with that change, once I get them to pass I'll resubmit.

Reply:
Great, sounds good. Thanks!